src/Project.php

```diff
@@ -196,6 +197,15 @@ private function getProjectName($path)
             return trim(basename($path, '.sln'));
         }
 
+        if (is_readable($path . '/app/src/main/res/values/strings.xml')) {
```

Comment:
You should convert `$path . '/app/src/main/res/values/strings.xml'` so you can reuse it. It will make it easier to read.

Reply:
yes, I will and for all other case also (planned before stable release)